packages/cli/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "../../configurations/tsconfig.json",
```

Comment:
The Typescript configuration that all the projects will use is extracted under `configurations` so that all the packages are consistently configured.